### DISCUSSION TITLE
Remove file from System game list

### DISCUSSION THREAD

**Discussion by dem1980** [Rparadise-Team/Koriki#31] [Ideas & Requests]
Hi,
it could be nice to be able to delete a rom file from the system menu.
For example i put some roms to test some hacks in my \ROMS\SFC directory.
Then i navigate into Koriki with my Myioo and in rom's list for the systeme SFC, I would like to open the popup menu and remove the file.
It could be easier than going into Apps and open the Commander to delete files.
Thank you

**Comment by albgr1979** [comment]
If you want to delete a game from the SimpleMenu gamelist, you can do it. The controls are explained on the wiki. By pressing the B + X buttons you do it.
